src/assemble_workflow/bundle_rpm.py

```diff
@@ -94,8 +97,9 @@ def build(self, name: str, dest: str, archive_path: str) -> None:
             [
                 'rpmbuild',
                 '-bb',
-                '--define',
-                f"'_topdir {ext_dest}'",
+                f"--define '_topdir {ext_dest}'",
+                f"--define '_version {build_cls.version}'",
+                f"--define '_architecture_alt {rpm_architecture(build_cls.architecture)}'",
```

Comment:
This looks like a bug, you should be passing `_architecture` into it.

Reply:
Fixed in https://github.com/opensearch-project/opensearch-build/pull/1829